hudi-utils/pom.xml

```diff
@@ -46,32 +46,62 @@
             <groupId>org.apache.hudi</groupId>
             <artifactId>hudi-common</artifactId>
             <version>${hudi.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.hbase</groupId>
```

Comment:
@yihua  I added exclusions. I had some issue with these jars during maven compile.

Reply:
intermittedn issue. Fixed and updated the pr. these exclusions are not needed.